### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by Young-W1**
Update the game over message so people know how to play again!

### CODE REVIEW

**Comment by Young-W1** [comment]
@Ph-uad adding you as a reviewer.

**Comment by Ph-uad** [comment]
ehn ehn

**Comment by Ph-uad** [comment]
> @Ph-uad adding you as a reviewer.

ehn ehn

**Review by Ph-uad — APPROVED** [review]
approved

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (Young-W1) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment (Young-W1) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again . Okay?🧑‍💻 🤖!</h2>
```

**Pull request merged**